content/contact/index.md

```diff
@@ -30,9 +30,9 @@ We welcome vulnerability reports [according to our vulnerability disclosure poli
 
 ### Questions from the public and industry
 
-If you have a question that isn't on behalf of a U.S. government organization (such as if you're a member of the public or representing a company), we invite you to post your question [publicly as an issue on GitHub]({{site.github_url}}/issues/new) (requires a free GitHub account), so that we can write an answer available to everyone. If you prefer not to post publicly, you can email [{{site.email}}](mailto:{{site.email}}).
+If you have a question that isn't on behalf of a U.S. government organization (such as if you're a member of the public or representing a company), we invite you to post your question [publicly as an issue on GitHub](https://github.com/cloud-gov/site/issues/new) (requires a free GitHub account), so that we can write an answer available to everyone. If you prefer not to post publicly, you can email [{{site.email}}](mailto:{{site.email}}).
 
-If you are part of a company asking a question to help prepare a response to a RFI/RFQ/RFP (or a similar proposal process), we will ask you to post your question [publicly as an issue on GitHub]({{site.github_url}}/issues/new) (requires a free GitHub account). We want to answer publicly so that our answer is available to all companies competing in that process.
+If you are part of a company asking a question to help prepare a response to a RFI/RFQ/RFP (or a similar proposal process), we will ask you to post your question [publicly as an issue on GitHub](https://github.com/cloud-gov/site/issues/new) (requires a free GitHub account). We want to answer publicly so that our answer is available to all companies competing in that process.
```

Comment:
```suggestion
If you are part of a company asking a question to help prepare a response to a RFI/RFQ/RFP (or a similar proposal process), we will ask you to post your question [publicly as an issue on GitHub]({{site.github_url}}/issues/new) (requires a free GitHub account). We want to answer publicly so that our answer is available to all companies competing in that process.
```